pridepy/tests/test_iprox_aspera.py

```
@@ -0,0 +1,300 @@
+             patch("pridepy.download.iprox.subprocess.run") as run, \
+             patch.object(IproxProvider, "_ascp_binary", return_value="/bin/ascp"):
+            run.return_value = MagicMock(returncode=0)
+            IproxProvider.aspera_download(
+                urls=[url],
+                output_folder=tmp,
+                relative_paths=["../../etc/x"],
+                user="bob",
+                password="secret",
+                maximum_bandwidth="100M",
+            )
+        args, kwargs = run.call_args
```

Ruff (0.15.20)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)